crates/holochain/src/conductor/chc/chc_local.rs

```diff
@@ -196,7 +196,7 @@ mod tests {
         let (agent, _) = SweetAgents::alice_and_bob();
 
         let (cell,) = conductor
-            .setup_app_for_agent("app", agent.clone(), [&dna_file])
+            .setup_app_for_agent("app", agent.clone(), &[dna_file])
```

Comment:
I could probably get these to all be consistent, but somehow in the mad dash to get everything working, the required syntax for these switched in both directions in some places...hmm...